03-path-application-development/310-chaos-engineering/readme.adoc

```diff
@@ -50,8 +50,6 @@ These services are built as Docker images and deployed in Kubernetes. All servic
 
 To deploy the application so that it can be a subject of a chaos experiment, execute the following:
 
-. Deploy the application:
-
   $ kubectl create -f templates/app.yml
```

Comment:
`app.yml` was removed. Shouldn't this command be using `template.yml`?

Also to be consistent, can you change the file extension to `yaml` instead of `yml`?

Reply:
At the moment, following other chapters, the file is still `app.yml` inside the `templates` folder. The command could also be shortened to `kubectl create -f templates` though?

No problem at all on changing the file extension, will do that now.

Reply:
Both changes now in, anything else needed?